tests/e2e/monovertex-rider-e2e/monovertex_rider_test.go

```diff
@@ -185,13 +185,17 @@ var _ = Describe("Rider E2E", Serial, func() {
 			return rollout, nil
 		})
 
-		// verify ConfigMap is created
+		monoVertexIndex++
```

Comment:
Now the volumeMounts change to the spec causes there to be a progressive rollout, so we need to check the new index